### PR TITLE
Plot data

### PR DESCRIPTION
**Pull request by manycoding**

### CODE REVIEW

**Comment by manycoding** [comment]
I uploaded the branch and the notebook so you can see the changes https://jupyter.scrapinghub.com/user/valery/lab/tree/shared/qa/Experiments/plot_data.ipynb, but I have to figure how to import the code from the folder.

**Comment by alexander-matsievsky** [comment]
@manycoding Somewhat unrelated to the PR at hand but what do you think about switching to Markdown for reporting, Valery?

image

File: setup.py [hunk 1]
```diff
@@ -12,17 +12,12 @@
...
 
 
-tests_require = [
-    "pytest",
-    "pytest-mock",
-    "pytest-cov",
-    "pytest-pythonpath",
-    "ipywidgets",
-]
+tests_require = ["pytest", "pytest-mock", "pytest-cov", "pytest-pythonpath"]
```
Comment (victor-torres) [inline setup.py]: I do prefer it as before: one item per line. 

Seems like it's easier to maintain and when adding/removing items, the diffs look smaller.

File: src/arche/arche.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from arche.readers.items import CollectionItems, JobItems
 import arche.readers.schema as sr
 from arche.report import Report
-import arche.rules.category_coverage as category_coverage
+import arche.rules.category as category_rules
```
Comment (victor-torres) [inline src/arche/arche.py]: Maybe `import arche.rules.categories as category_rule` (?)

File: src/arche/rules/coverage.py [hunk 3]
```diff
@@ -1,76 +1,73 @@
...
+    fields_coverage.name = "Fields Coverage"
 
-    empty_fields = fields_coverage[fields_coverage["Values Count"] == 0]
-    result_msg = f"{len(empty_fields)} totally empty field(s)"
+    empty_fields = fields_coverage[fields_coverage == 0]
 
     result = Result("Fields Coverage")
     if empty_fields.empty:
-        result.add_info(result_msg, detailed_msg)
+        result.add_info(f"PASSED", stats=fields_coverage)
```
Comment (victor-torres) [inline src/arche/rules/coverage.py]: No need for an f-string here.

**Comment by manycoding** [comment]
> @manycoding Somewhat unrelated to the PR at hand but what do you think about switching to Markdown for reporting, Valery?
I got back to this one. I would love to use markdown but it doesn't support coloring.

**Pull request merged**